src/diffusers/models/lora.py

```diff
@@ -156,15 +155,28 @@ def _unfuse_lora(self):
         self.w_up = None
         self.w_down = None
 
-    def forward(self, hidden_states, scale: float = 1.0):
+    def add_scale(self, scale: float = 1.0):
```

Comment:
This would break the `fuse` and `unfuse` operation. We cannot change `network_alpha` on the fly